cmd/http-check/main.go

```diff
@@ -29,30 +30,53 @@ func init() {
 }
 
 func main() {
-	log.Infoln("Beginning check.")
+	log.Println("Beginning check.")
+	checksRan := 0
+	checksPassed := 0
+	checksFailed := 0
+
+	ticker := time.NewTicker(time.Second)
```

Comment:
is it okay to spam 10 requests in 10 seconds to a URL? Would it get us blacklisted via robots.txt? https://support.google.com/webmasters/answer/6062608?hl=en

Reply:
i doubt 10 requests in 10 seconds would get anyone banned from anything, but if we find that to be true we can adjust.

It is probably prudent to break the tries and seconds of delay into environment variables.  @jdowni000 can you implement please?  Be sure to include the new environment variables you create in the example specs.

Reply:
I have added this logic with the latest commit @integrii